tests/system/action/speaker/test_update.py

```diff
@@ -26,8 +26,6 @@ def test_update_correct(self) -> None:
         )
         response = self.request("speaker.update", {"id": 890, "speech_state": "pro"})
         self.assert_status_code(response, 200)
-        model = self.get_model("speaker/890")
-        assert model.get("speech_state") == "pro"
```

Comment:
Please add this back (probably a rebase helps)

Reply:
Done.